postgis/postgis.sql.in

```diff
@@ -417,14 +423,16 @@ CREATE OPERATOR > (
 );
 
 -- Availability: 0.9.0
+-- Changed: 3.0.0
 CREATE OPERATOR CLASS btree_geometry_ops
```

Comment:
This will require dropping the old  operator on upgrade, right? I think dronnie is complaining about it.

Reply:
Made it the way it is in ND GiST, complaint gone.